api/api/settings/defaults.py

```diff
@@ -0,0 +1,131 @@
+"""
+Django settings for shareregistration project.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.7/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.7/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+
+from django.conf import global_settings
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '...'
```

Comment:
Do we need this? What is it for? Should it be a setting that is not version controlled?